src/e2e-test/resources/pluginParameters.properties

```diff
@@ -208,20 +208,26 @@ bqDataTypeTestFileSchema1=[{"key":"Id","value":"long"},{"key":"Value","value":"l
 TableKeyUpsert=ID
 TableKeyInsert=ID
 bqUpsertExpectedFile=testdata/BigQuery/BQUpsertTableFile
-bqUpdatededupeExpectedFile=testdata/BigQuery/BQUpdateDedupeFile
+bqUpdateDedupeExpectedFile=testdata/BigQuery/BQUpdateDedupeFile
 bqInsertExpectedFile=testdata/BigQuery/BQInsertIntFile
 relationTableKeyValue=Name
 dedupeByOrder=ASC
 dedupeByValue=ID
+dedupeByValueUpsert=Price
 rangeStartValue=2
 rangeEndValue=3
 rangeIntervalValue=1
 partitionByFieldValue=ID
+bqPartitionFieldDateTime=transaction_dt
+bqPartitionFieldTimeStamp=updated_on
+bqSourceTable2=dummy
 dedupeBy=DESC
 TableKeyDedupe=Name
 Directive_Drop=testdata/BigQuery/test_diffschema_record-cdap-data-pipeline.json
 bqUpsertDedupeFile=testdata/BigQuery/BQUpsertDedupeFile
 bqDifferentRecordFile=testdata/BigQuery/BQDifferentRecordNameFile
+bqDateExpectedFile=testdata/BigQuery/BQTimeStampFile
```

Comment:
Use proper naming convention , Change the file name from 'BQTimeStampFile' to 'BQDateFile'

Reply:
Resolved  changed the name testdata/BigQuery/BQDateFile